contracts/SortitionPool.sol

```diff
@@ -35,15 +42,33 @@ contract SortitionPool is SortitionTree {
       _stakingContract,
       _minimumStake,
       _poolWeightDivisor,
-      _poolOwner
+      _poolOwner,
+      false
     );
   }
 
+  /// @notice Locks the sortition pool. In locked state, members cannot be
+  ///         inserted and removed from the pool. Members statuses cannot
+  ///         be updated as well.
+  /// @dev Can be called only by the contract owner.
+  function lock() public {
+    require(msg.sender == poolParams.owner, "Caller is not the owner");
```

Comment:
Can we take the opportunity and extract it to `onlyOwner` modifier?

Reply:
Done.